### ISSUE TITLE
Onboarding experience

### ISSUE THREAD

**Issue by sla-shi** [status-im/ideas#24]
## Preamble

    Idea: 24-onboarding
    Title: Onboarding experience
    Status: Draft
    Update: 2017-12-21
    Requires: #26 

## Summary
Status, Ethereum, cryptocurrency and the decentralized internet are new paradigms for many users. The job of onboarding will need to educate and inspire users about these new technologies, as well as help users get started with Status, create an account, send their first message, explore their first app, and initiate their first transaction.

Onboarding will be an important part of the upcoming [beta release](#26).  Currently this is in a preliminary explore phase, and the requirements will evolve as the beta work matures. 

This swarm will be the collaboration of marketing, product, design, and engineering to ensure that users have a consistent brand and product experience.

## Vision

In progress.

## Swarm Participants

- Lead Contributor: 
- Testing & Evaluation: @annadanchenko 
- Contributor: 
- Contributor: 
- UX: @andmironov
- PM: @chadyj
- Marketing: @j-zerah

Swarm channel: #24-onboarding

## Requirements

Create onboarding flow mockups 
As we define the brand tone of voice, make sure this is reflected in the on boarding experience.

### Minimum Viable Product

Goal Date:  
Description: 

### Iteration 1..N
Goal Date:  
Description: 

## Copyright
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).

**Comment by denis-sharypin** [comment]
![onboarding 02](https://user-images.githubusercontent.com/6559881/33017981-bf8500d2-cdf4-11e7-9ebf-d2a1d63eefaa.png)

To start a discussion, I have a general idea how to implement an onboarding that could satisfy the most of our [design principles (https://docs.google.com/document/d/1pKouXBAcMHVOySkSI1we4N-qhA9smLNbs5uW2QEdjaQ/edit). 

We can create screens like that on every new concept or a complicated UX for users. For instance, if a user sends a message to an offline user we show him a 'blue screen of help' saying that Status is a p2p messenger and works differently.

I like this approach because it's flexible and has a simple logic behind. UX flow is like that: 
1. a user is going to make an action that requires SNT; 
2. we are checking was a concept of SNT introduced to the user;
3. if not, we simply show him a help screen with the description of what SNT is.

Main disadvantages of this modular onboarding are: sometimes a user can see two screens in a row like I had it here https://github.com/status-im/ideas/pull/23 or this approach could feel like braking and interrapting main userflows.

**Comment by andmironov** [comment]
I like the idea of the "blue screen of help", it would be easy to integrate it in any flow and i like the idea of it being colourful so that it would contrast with the rest of the UI. showing and explaining a certain aspect only when it's relevant, but let's think about people who want to get the whole picture at once.

**Comment by andmironov** [comment]
I guess, after all we might consider using a different background colour not to confuse the "screen of help" with wallet.

**Comment by mcivor25** [comment]
This is looking pretty nice :) I would consider Gamification, during your on-boarding process. Use some gestures and have some fun with it. Clicking accept every time gets repetitive and boring. Add a 'swipe to accept' feature when confirming the passphrase. Makes things memorable :) 

Since you are giving 10 ETH credits to learn, why not have this as a 'reward' for completing the tutorial. This would help newcomers to understand Cryptocurrency and your process quicker!

I would agree with your different color background too, white would be super clean and allow for you to customize interactions and animations with the gradient colors on the Status landing page.

**Comment by flexsurfer** [comment]
want to contribute ~20h/w

**Comment by mcivor25** [comment]
Is this an open offer or you have to work at Status? :)

**Comment by chadyj** [comment]
Onboarding will be an important part of the upcoming beta release. Currently this is in a preliminary explore phase, and the requirements will evolve as the beta work matures.

This swarm is the collaboration of marketing, product, design, and engineering to ensure that users have a consistent brand and product experience.

I updated the issue description to reflect this.

**Comment by annadanchenko** [comment]
What if we build a separate DApp to put all the educational content in it? This would be a separate piece to be built but outside of the Status app. In the same time "it knows" your address, so if we would add Gamification to it, as @mcivor25 suggested, then it still can be done, so for completing a tutorial user can get some bonus... @andmironov/@flexsurfer would you be interested in designing/building such DApp?

**Comment by oskarth** [comment]
Closing in favor of the more high-resolution/texture https://github.com/status-im/ideas/issues/80 which completely supersedes this.
